xbmc/rendering/gles/RenderSystemGLES.h

```diff
@@ -12,11 +12,11 @@
 #include "rendering/RenderSystem.h"
 #include "utils/ColorUtils.h"
 
-#include <array>
+#include <map>
 
 #include "system_gl.h"
 
-enum ESHADERMETHOD
```

Comment:
shouldn't enums have camelcase-style names?

Reply:
Actually PascalCase https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#83-enums

I can update it.